packages/reapit-connect-management/src/components/ui/offices/__tests__/create-office-group.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { notification } from '@reapit/elements'
+import CreateOfficeGroupModal, { CreateOfficeGroupModalProps, onHandleSubmit } from '../create-office-group'
+
+const filterProps = (): CreateOfficeGroupModalProps => ({
+  visible: true,
+  setOpenCreateGroupModal: jest.fn(),
+  orgId: '1185e436-3b7e-4f67-a4b7-68f83054ad3c',
+  onRefetchData: jest.fn(),
+})
+
+jest.mock('../../../../core/connect-session')
+const mockResponse = 'success'
+
+const mockFetchPromise = Promise.resolve({
+  json: () => mockResponse,
+})
+
+describe('CreateOfficeGroupModal', () => {
+  it('should match a snapshot', () => {
+    expect(shallow(<CreateOfficeGroupModal {...filterProps()} />)).toMatchSnapshot()
+  })
+})
+
+describe('onHandleSubmit', () => {
+  const name = 'Group name'
+  const handleOnClose = jest.fn()
+  const onRefetchData = jest.fn()
+  const orgId = 'ORG1'
+  const officeIds = ['OF1', 'OF2']
+  const onSubmit = onHandleSubmit(handleOnClose, onRefetchData, orgId)
+
+  it('should show notification error', async () => {
+    window.fetch = jest.fn().mockImplementation(() => mockFetchPromise)
```

Comment:
Window.fetch is already mocked by default in the jest setup.

You can do 

`(window.fetch as jest.Mock).mockImplementation...`

It's fine for now though!

Reply:
we used jest-fetch-mock already, nice!